enable/disable caching only after preloading function is called. Remo… (#72)
Shopify/checkout-kit-swift: README.md

Initializing a checkout session requires communicating with Shopify servers and, depending on the network weather and the quality of the buyer's connection, can result in undesirable waiting time for the buyer. To help optimize and deliver the best experience, the SDK provides a preloading hint that allows app developers to signal and initialize the checkout session in the background and ahead of time.

Preloading is an advanced feature that will only be activated after calling:
```swift
ShopifyCheckoutKit.preload(checkout: checkoutURL)
```

> Once preload is called, the checkout view is cached. You are expected to re-call preload every time you need to refresh the checkout view. If for whatever reason you need to invalidate the cache without calling checkout again, you can call this function:

Author: We couldn't think of any outside our test sample app use case

> Once preload is called, the checkout view is cached. You are expected to re-call preload every time you need to refresh the checkout view. If for whatever reason you need to invalidate the cache without calling checkout again, you can call this function:

Author: I documented the test case scenario!

```swift

> ShopifyCheckoutKit.configuration.preloading.invalidateAllCaches()

Author: No, just being explicit

> ShopifyCheckoutKit.configuration.preloading.invalidateAllCaches()

Reviewer: What about `.clearCache()`?

```

**Important considerations:**

> 1. Once you call preload, the checkout view is *cached*. _you must remember to call preload again on every cart update._ Otherwise you risk the user checking out with a stale cart

Reviewer: ```suggestion
1. Once you call `preload()`, the checkout view will be **cached**. **You must call `preload()` again on every cart update**. Otherwise you risk the user checking out with a stale cart.
```
